setup.py

```diff
@@ -77,6 +77,8 @@ def get_long_desc():
         "PyYAML>=5.0",
         "tqdm>=4.0",
         "wwt_data_formats>=0.10.2",
+        "shapely",
+        "reproject",
```

Comment:
Whenever I have a list of things, I try to keep it alphabetized if it doesn't have some other inherent ordering.